docs/kyma/docs/04-03-cluster-installation.md

```diff
@@ -0,0 +1,518 @@
+    export DNS_ZONE={YOUR_DNS_ZONE}
+    ```
+
+2. To add DNS entries, run these commands:
+    ```
+    export EXTERNAL_PUBLIC_IP=$(kubectl get service -n istio-system istio-ingressgateway -o jsonpath="{.status.loadBalancer.ingress[0].ip}")
+
+    export APISERVER_PUBLIC_IP=$(kubectl get service -n kyma-system apiserver-proxy-ssl -o jsonpath="{.status.loadBalancer.ingress[0].ip}")
+
+    export REMOTE_ENV_IP=$(kubectl get service -n kyma-system application-connector-ingress-nginx-ingress-controller -o jsonpath="{.status.loadBalancer.ingress[0].ip}")
+
+    gcloud dns --project=$PROJECT record-sets transaction start --zone=$DNS_ZONE
+
+    gcloud dns --project=$PROJECT record-sets transaction add $EXTERNAL_PUBLIC_IP --name=\*.$DOMAIN. --ttl=60 --type=A --zone=$DNS_ZONE
+
+    gcloud dns --project=$PROJECT record-sets transaction add $REMOTE_ENV_IP --name=\gateway.$DOMAIN. --ttl=60 --type=A --zone=$DNS_ZONE
+
+    gcloud dns --project=$PROJECT record-sets transaction add $APISERVER_PUBLIC_IP --name=\apiserver.$DOMAIN. --ttl=60 --type=A --zone=$DNS_ZONE
+
+    gcloud dns --project=$PROJECT record-sets transaction execute --zone=$DNS_ZONE
+    ```
+  </details>
+  <details>
+  <summary>
+  AKS
+  </summary>
+
+This Installation guide shows developers how to quickly deploy Kyma on an [Azure Kubernetes Service](https://azure.microsoft.com/services/kubernetes-service/) (AKS) cluster.
+
+## DNS setup and TLS certificate generation (optional)
+
+>**NOTE:** Execute instructions from this section only if you want to use your own domain. Otherwise, proceed to [this](#installation-install-kyma-on-a-gke-cluster-prepare-the-installation-configuration-file-1) section.
+
+### Delegate the management of your domain to Azure DNS
+
+Follow these steps:
+
+1. Export the domain name, and sub-domain as environment variables. Run the commands listed below:
+
+    ```
+    export DNS_DOMAIN={YOUR_DOMAIN} # example.com
+    export SUB_DOMAIN={YOUR_SUBDOMAIN} # cluster (in this case the full name of your cluster is cluster.example.com)
+    ```
+
+2. Create a DNS-managed zone in your Azure subscription. Run:
+
+    ```
+    az network dns zone create -g $RS_GROUP -n $DNS_DOMAIN
+    ```
+
+    Alternatively, create it through the Azure UI. In the **Networking** section, go to **All services**, click **DNS zones**, and select **Add**.
+
+3. Delegate your domain to Azure name servers.
+
+    - Get the list of the name servers from the zone details. This is a sample list:
+      ```
+      ns1-05.azure-dns.com.
+      ns2-05.azure-dns.net.
+      ns3-05.azure-dns.org.
+      ns4-05.azure-dns.info.
+      ```
+
+    - Set up your domain to use these name servers.
+
+4. Check if everything is set up correctly and your domain is managed by Azure name servers. Run:
+    ```
+    host -t ns $DNS_DOMAIN
+    ```
+    A successful response returns the list of the name servers you fetched from Azure.
+
+## Get the TLS certificate
```

Comment:
I think this heading should be H3 instead of H2. It's part of `## DNS setup and TLS certificate generation (optional)`, isn't it?